components/src/core/components/Input/checkbox-input.scss

```diff
@@ -128,3 +127,7 @@
     align-self: center;
   }
 }
+
+span.oxd-checkbox-input[tooltip]::after {
+  max-width: 150px;
```

Comment:
why this is needed?

Reply:
UI team wanted to decrease the width of the tooltip